tests/cross/src/main/scala/tests/pc/BaseTypeDefinitionSuite.scala

```diff
@@ -0,0 +1,27 @@
+package tests.pc
+
+import tests.BasePCSuite
+import scala.concurrent.duration.Duration
+
+class BaseTypeDefinitionSuite extends BasePCSuite {
+  val test: (String, String) => (String, String) =
```

Comment:
Can we rename this to to something more meaningful?

Reply:
Changed.